pkg/controller/apimanager/apimanager_controller.go

```diff
@@ -100,11 +104,50 @@ type ReconcileAPIManager struct {
 	operator.BaseControllerReconciler
 }
 
+const (
+	ThreescaleVersionAnnotation = "apps.3scale.net/apimanager-threescale-version"
+	OperatorVersionAnnotation   = "apps.3scale.net/threescale-operator-version"
+)
+
+func (r *ReconcileAPIManager) updateVersionAnnotations(cr *appsv1alpha1.APIManager) error {
+	if cr.Annotations == nil {
```

Comment:
Annotations can be nil?

Reply:
Yes it seems so.

When you don't have annotations you have the zero value of them. Annotations is a map[string]string. The zero-value of a map in Go is nil.

Reply:
:+1: